exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/MetricAdapter.java

```diff
@@ -189,6 +194,8 @@ private static int estimateNumSamples(int numPoints, MetricDataType type) {
         return metricData.getLongSumData().getPoints();
       case SUMMARY:
         return metricData.getDoubleSummaryData().getPoints();
+      case HISTOGRAM:
```

Comment:
Call this DOUBLE_HISTOGRAM?

Reply:
I think we will come back to this PR after #2581 is done? sorry for the confusion.